### ISSUE TITLE
EXP Vanishes when switching dimensions using the Nether Pet/Home Pet

### ISSUE THREAD

**Issue by ghost** [capnkirok/Inventory-Pets#553]
Title, I tested an enchantment table and it is not just visual. EXP is restored when you gain an EXP point.

**Comment by ghost** [comment]
Minecraft/Forge 1.12.2, forgot to mention.

**Comment by capnkirok** [comment]
When you say XP is restored once you gather another XP ... is this just a visual bug?

**Comment by ghost** [comment]
> When you say XP is restored once you gather another XP ... is this just a visual bug?

Nope, i checked enchantment table and it did nothing, not enough EXP.
